tox.ini

```diff
@@ -19,7 +19,7 @@ commands =
     mypy: mypy --python-version 3.5 uaclient/
     mypy: mypy --python-version 3.6 uaclient/
     mypy: mypy --python-version 3.7 uaclient/
-    black: black --check --diff uaclient/
```

Comment:
Why?

(This should at least be a separate commit capturing the rationale.)

Reply:
bogus commit. removed and forgot to push.